### DISCUSSION TITLE
scammer

### DISCUSSION THREAD

**Discussion by Alimayouf** [community/community#33439] [Other Feature Feedback, Questions, & Ideas] [answered]
Fnoberz this man is a scam he took my money for service and he block me from discord

**Comment by samuel-ma** [comment]
@Alimayouf 
Reach out, might help you out
I've been that guy before

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Alimayouf 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
